### ISSUE TITLE
Can't upgrade to latest version (2.5.3) on ubuntu 20.04

### ISSUE THREAD

**Issue by narudarurarasya** [GothenburgBitFactory/taskwarrior#2433]
Hello,

I have trouble upgrading to latest release on my ubuntu 20.04 LTS machine. It says the following...

```
❯ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
❯ task --version
2.5.1
❯ sudo apt-get upgrade taskwarrior
Reading package lists... Done
Building dependency tree       
Reading state information... Done
taskwarrior is already the newest version (2.5.1+dfsg-9).
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```

Am I missing anything? Any ideas on how do I upgrade to `2.5.3` without cloning repo or re-installing latest release manually? :)

Cheers

**Comment by tbabej** [comment]
Hi @narudarurarasya,

we don't maintain the distribution packages directly, so this would be a question for Ubuntu/Debian package maintainers.

From the [package overview listed here], however, I can see that `2.5.1` is indeed the highest version available in 20.10 and `2.5.3` will be available in `21.04 (Hirsute Hippo)`.

**Comment by narudarurarasya** [comment]
thanks a lot for clearning that up @tbabej  :)
